languages/python/django-oso/django_oso/partial.py

```diff
@@ -76,11 +79,10 @@ def isa_expr(expr: Expression, model: Model, **kwargs):
         model = getattr(model, attr).field.related_model
     constraint_type = apps.get_model(django_model_name(right.tag))
     assert not right.fields, "Unexpected fields in matches expression"
-    return None if issubclass(model, constraint_type) else FALSE_FILTER
+    return TRUE_FILTER if issubclass(model, constraint_type) else FALSE_FILTER
 
 
 def and_expr(expr: Expression, model: Model, **kwargs):
-    assert expr.operator == "And"
```

Comment:
Why remove these assertions? They are there since we don't have typed expressions for each operation. This ensures the input to this function is valid.

Reply:
Didn't seem useful since we're already switching on the operator in `translate_expr()`, but I don't really mind them and have added them back in a3447049